component/admin/controllers/icalevent.php

```diff
@@ -1182,13 +1182,13 @@ private function doSave(& $msg)
 		if ($event = SaveIcalEvent::save($array, $this->queryModel, $rrule))
 		{
 			$row = new jIcalEventRepeat($event);
-			if (JEVHelper::canPublishEvent($row))
+			if (!JEVHelper::canPublishEvent($row) && !$event->state)
```

Comment:
My missreading !  I need stronger glasses I think, I didn't spot the ! at the start.